includes/internal/emails/class-email-list-table.php

```diff
@@ -264,4 +283,32 @@ public function get_bulk_actions() {
 			'bulk-enable-email'  => __( 'Enable', 'sensei-lms' ),
 		];
 	}
+
+	/**
+	 * Check if the email is available.
+	 *
+	 * @param \WP_Post $post The email post.
+	 *
+	 * @internal
+	 * @access private
+	 *
+	 * @return boolean True if the email is available, false otherwise.
+	 */
+	public function is_email_available( $post ) {
+		$available = ! get_post_meta( $post->ID, '_sensei_email_is_pro', true );
+
+		/**
+		 * Filter if the email is available.
+		 *
+		 * @since $$next-version$$
+		 * @hook sensei_email_is_available
+		 *
+		 * @param {boolean} $available True if the email is available, false otherwise.
+		 * @param {object}  $post The post.
+		 * @param {object}  $list_table Email_List_Table instance.
+		 *
+		 * @return {boolean}
+		 */
+		return apply_filters( 'sensei_email_is_available', $available, $post, $this );
```

Comment:
Very interesting because it will only work when the user install the latest sensei version 👍